rust/main/Cargo.toml

```diff
@@ -32,7 +32,7 @@ anyhow = "1.0"
 async-trait = "0.1"
 async-rwlock = "1.3"
 auto_impl = "1.0"
-axum = "0.6.1"
+axum = { version = "0.6.1", features = ["macros"] }
```

Comment:
Can remove this feature if we deem it unnecessary and makes the production binary too big.
I just found it useful during development to have more clear error messages.
See https://docs.rs/axum/latest/axum/attr.debug_handler.html

Reply:
nicee it was actually quite frustrating to debug axum problems but i'd keep this feature as a dev dependency if possible pls - it doesn't help the release build once it passes e2e

Reply:
Updated to dev-dependency